bezier/src/line.rs

```diff
@@ -57,10 +57,14 @@ impl LineSegment {
         LineSegment { from: self.sample(t), to: self.to }
     }
 
-    /// [Not implemented]
+    /// Return the minimum bounding rectangle
     #[inline]
     pub fn bounding_rect(&self) -> Rect {
-        unimplemented!()
+        let width = (self.from.x - self.to.x).abs();
+        let mid_x = self.from.x.min(self.to.x) + width / 2.;
```

Comment:
Looks good except one thing: Rect is defined by its top-left corner (minimum x and y) and a size. In the two implementation it appears that you created the rectangle using its mid point instead of the origin.